Duffel.ApiClient/DuffelApiClient.cs

```diff
@@ -22,8 +23,10 @@ public class DuffelApiClient : IDuffelApiClient
         
         public DuffelApiClient(string accessToken, bool production = false)
         {
+            var executingAssemblyName = Assembly.GetExecutingAssembly().GetName();
             _httpClient.BaseAddress = production ? new Uri("https://api.duffel.com") : new Uri("https://api.staging.duffel.com");
-
+            
+            _httpClient.DefaultRequestHeaders.Add("User-Agent", $".net.{executingAssemblyName.Name}/{executingAssemblyName.Version}");
```

Comment:
The value of the "User-Agent" HTTP header should look like `Duffel/:api-version duffel_api_:language/:client-sdk-version`, e.g. `Duffel/beta duffel_api_dotnet/0.0.3-alpha`. I'm not that familiar with .NET so can't tell if this will render to match that.

Reply:
🙇 fixed!